drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-testing/Readme.md

```diff
@@ -0,0 +1,16 @@
+Kogito Testing (Development) Webapp
+==================================
+
+This webapp is meant as testing/developing environment. It **provides** buttons, methods and functions (ex. file retrieval) that should be provided by running environment (ex. vs code).
+
+Testing
+--------------
+1. full compilation (i.e. included GWT compilation) of the testing module
```

Comment:
full compilation - Full compilation

Reply:
Are you sure you want to keep it as Readme.md not a README.md

Reply:
@dupliaka Updated :)